docs/appkit/next/solana/hooks.mdx

```diff
@@ -51,7 +51,7 @@ function SignMessage() {
 ### useAppKitError
 
 ```ts
-import { useAppKitError } from '@web3modal/solana/react'
+import { useAppKitError } from '@reown/solana/react'
```

Comment:
I don't think this is the correct import structure. Can you please check if this works locally with your code?